lib/index.js

```diff
@@ -9579,25 +9579,27 @@ var external_child_process_ = __nccwpck_require__(3129);
  */
 const execShellCommand = (cmd) => {
   return new Promise((resolve, reject) => {
-    const process = global.process.platform !== "win32" ?
+    const proc = process.platform !== "win32" ?
```

Comment:
In general, I prefer separating benign renames into their own commits, to make reviewing more focused (on concern per commit).

Just a slight preference, not a blocker.

Reply:
Good point! Will try to do that in the future, depends always on the reviewer if he reviews by commits or against the whole change set.